app/routes/event-invoice/review.js

```diff
@@ -1,21 +1,23 @@
 import Route from '@ember/routing/route';
 
 export default class extends Route {
-  async model(params) {
-    let filterOptions = [
-      {
-        name : 'identifier',
-        op   : 'eq',
-        val  : params.invoice_identifier
-      }
-    ];
-    return {
-      'user' : await this.authManager.currentUser,
-      'data' : await this.store.query('event-invoice', {
-        include : 'event',
-        filter  : filterOptions
-      })
-    };
+  titleToken(model) {
+    let invoice_identifier = model.get('identifier');
+    return this.l10n.t(`Review Event Invoice -${invoice_identifier}`);
```

Comment:
tVar, and can't the variable be eliminated? it's being called just once.

Reply:
@CosmicCoder96 eliminated the variable